pkg/session_recording/recordings.jsx

```diff
@@ -193,13 +217,77 @@
             });
         }
 
+        sendToTerm(value) {
+            this.refs.terminal.state.terminal.send(value);
+        }
+
+        playPauseToggle() {
+            if (this.state.playback === 'pause') {
+                this.setState({playback: 'play'});
+            } else {
+                this.setState({playback: 'pause'});
+            }
+            this.sendToTerm(' ');
+        }
+
+        speedUp() {
+            if (this.state.speed < 4) {
+                let speed = this.state.speed + 1;
+                this.setState({speed: speed});
+                this.sendToTerm('}');
+            }
+        }
+
+        speedDown() {
+            if (this.state.speed > -4) {
+                let speed = this.state.speed - 1;
+                this.setState({speed: speed});
+                this.sendToTerm('{');
+            }
+        }
+
+        speedReset() {
+            this.setState({speed: 0});
+            // Backspace
+            this.sendToTerm('\x7f');
+        }
+
         restartPlayback() {
             if (this.state.channel != null) {
                 this.state.channel.close();
             }
             this.setState({channel: this.createChannel()});
         }
 
+        endOnce() {
```

Comment:
Perhaps something like `ffToEnd`, or `fastForwardToEnd` will be a better name for this function. What do you think?